src/jquery.floatThead.js

```diff
@@ -95,6 +95,10 @@
         return result;
       };
     };
+    that.isFunction = function(obj) {
```

Comment:
are you sure you didnt have your own local version of `window._` without that method?

does this code not do what I think it does?

```js
$.each(isThings, function(){
       var name = this;
       that['is' + name] = function(obj) {
         return Object.prototype.toString.call(obj) == '[object ' + name + ']';
       };
     });
```